rpm/clang.spec

```diff
@@ -153,7 +153,7 @@ pushd build
 %endif
 	-DCLANG_INCLUDE_TESTS:BOOL=OFF \
 	-DLLVM_MAIN_SRC_DIR=%{_datadir}/llvm/src \
-%if 0%{?__isa_bits} == 64
+%ifarch aarch64
```

Comment:
Why this change? This limits the usage for only aarch64 ignoring for example x86_64, I would prefer using the old way unless there are some issues with that?

Reply:
@direc85 do you know why you pushed this?

Reply:
If __isa_bits doesn't work you can also compare against %{_lib}.

Reply:
This was due to not clearing the previous build folder before changing architecture, so that change can be reverted.